app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="devise_login_register">
+    <%= f.text_field :name, placeholder: "Full Name", autofocus: true %>
+  </div>
+
+  <div class="devise_login_register">
+    <%= f.email_field :email, autocomplete: "email", placeholder: :email%>
+  </div>
+
+  <div class="devise_login_register">
+    <%= f.password_field :password, autocomplete: "new-password", placeholder: :password %>
+  </div>
+
+  <%# <div class="field"> %>
+    <%# <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: :password_confirmation %>
+  <%# </div> %>
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
@BenMukebo You are absolutely right, thank you very much for your comments. I have already applied your comments and applied your advice.

Reply:
This change has been done well